models/offer.js

```diff
@@ -18,46 +18,46 @@ const offerSchema = new Schema(
       type: [String],
       enum: {
         values: PROFICIENCY_LEVEL_ENUM,
-        message: `Proficiency level can be either of these: ${PROFICIENCY_LEVEL_ENUM.toString()}`
+        message: ENUM_CAN_BE_ONE_OF('proficiency level', PROFICIENCY_LEVEL_ENUM)
       },
       required: [true, FIELD_CANNOT_BE_EMPTY('proficiency level')]
     },
     description: {
       type: String,
-      minlength: [1, 'Description cannot be shorter than 1 symbol.'],
-      maxlength: [200, 'Description cannot be longer than 300 symbols.'],
```

Comment:
why did you change it?
there are minlength and maxlength in mongoose, you dont need camelCase in here

Reply:
i know, but for me it's looking better and more readable